addons/web/static/src/core/effects/rainbow_man.scss

```diff
@@ -1,256 +1,156 @@
 .o_reward {
-  $reward-size: 400px;
-  $reward-size-mobile: 300px;
-  $reward-text-color: #727880;
-  $reward-base-time: 1.4s;
+    $reward-base-time: 1.4s;
 
-  will-change: transform;
-  z-index: $zindex-modal;
-  padding: 50px;
-  margin: -5% auto 0 (-$reward-size / 2);
-  @include media-breakpoint-down(sm) {
-    margin: -5% auto 0 (-$reward-size-mobile / 2);
-  }
-  background-image: -webkit-radial-gradient(#edeff4 30%, transparent 70%, transparent);
-  background-image: -o-radial-gradient(#edeff4 30%, transparent 70%, transparent);
-  background-image: radial-gradient(#edeff4 30%, transparent 70%, transparent);
-
-  animation: reward-fading ($reward-base-time * 0.5) ease-in-out 0s 1 normal forwards;
-  @include o-position-absolute(20%, auto, auto, 50%);
-  @include size($reward-size, $reward-size);
-  @include media-breakpoint-down(sm) {
-    @include size($reward-size-mobile, $reward-size-mobile);
-  }
-
-  &.o_reward_fading {
-    display: block;
-    animation: reward-fading-reverse ($reward-base-time * 0.4) ease-in-out 0s 1 normal forwards;
+    inset: 0; // Same than : left: 0; right: 0; top: 0; bottom: 0;
+    will-change: transform;
+    z-index: $zindex-modal;
+    animation: reward-fading $reward-base-time * 0.5 ease-in-out forwards;
 
-    .o_reward_face_group {
-      animation: reward-jump-reverse ($reward-base-time * 0.4) ease-in-out 0s 1 normal forwards;
+    .o_reward_box {
+        transform-box: fill-box;
     }
-
-    .o_reward_rainbow {
-      path {
-        animation: reward-rainbow-reverse ($reward-base-time * 0.5) ease-out 0s 1 normal forwards;
-      }
+    &.o_reward_fading {
```

Comment:
Add an empty line before ` &.o_reward_fading`... the same for all the other selectors